queries.sql

```diff
@@ -1,17 +1,58 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';
+BEGIN TRANSACTION;
 
-SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+  UPDATE animals SET species='unspecified';
 
-SELECT name FROM animals WHERE neutered=true AND escape_attempts<3;
+  SELECT * FROM animals ORDER BY id; 
 
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+ROLLBACK;
```

Comment:
- [ ] It is a good practice to always verify your changes after rolling back a transaction.
In that case, please add a statement to select all species from animals.

```sql
SELECT species from animals;
```

Reply:
> * [x]  It is a good practice to always verify your changes after rolling back a transaction.
>   In that case, please add a statement to select all species from animals.
> 
> ```sql
> SELECT species from animals;
> ```

- [ ] As the previous reviewer suggested, Please work on it. :+1: